### ISSUE TITLE
Feature request: MyMovies.dk to Trakt collection

### ISSUE THREAD

**Issue by EffeF** [damienhaynes/TraktRater#68]
I would like to be able to add my collection of DVD and Blu-Ray to the Trakt collection.

The collection is exportable from My Movies in .CSV format:
`collection number; Disc Title; tittle; barcode; type; year; IMDB id; duration; film director; cast`
eg:
`125; Disc Title; Deadpool; 8010312120428; Blu-ray; 2016; tt1431045; 108; Tim Miller; Ryan Reynolds,Karan Soni,Ed Skrein,Michael Benyaer,Stefan Kapicic,Brianna Hildebrand,Style Dayne,Kyle Cassie,Taylor Hickson,Ayzee,Tim Miller,Rhett Reese,Paul Wernick,Geoff Johns,Rob Liefeld,Len Wein,Dave Cockrum,Fabian Nicieza,Grant Morrison,Joe Kelly,Ed McGuinness,Frank Quitely,Joe Madureira`

or it can be exported in .txt format:
`title - type, year`
eg:
`Deadpool - Blu-ray, 2016`

you can add support to My Movies?

**Comment by damienhaynes** [comment]
It sounds possible, I will look into it.

Out of curiosity, how popular is that website for managing your collection? Do you think there will be much interest except for yourself for people to migrate to trakt?

**Comment by EffeF** [comment]
Yes, My Movies it's a very used service for managing collections, but I do not know how many people are interested.
